docs/source/release-notes/v86-k2.md

```diff
@@ -0,0 +1,104 @@
+* Add a "More" app that lists and installs community-made scripts and apps directly from a [GitHub repo](https://github.com/kasenvr/community-apps).
+* This takes longer than you might expect: Tons of revisions to brand/debrand, update links, docs, rehost, etc. in the Interface by various contributors. Thank you! (#9, #11, #19, #23, #34, #37, #38, #40, #44, #47, #50, #58, #70, #71, #75, #85, #89, #91, #99, #100, #105, #116, #177, #219)
+
+#### Graphics
+
+* Procedural shaders on mesh (Avatars, Models) support added. (#???)
+* Procedural vertex shader support added. (#15)
+* Procedural shaders no longer require an environment variable to be activated. Instead, use Developer > Render > Enable Procedural Materials. (#8)
+* Add support for GLTF alphaModes. (#15)
+* Fix opacityMapModes for GLTF. (#15)
+</br><img src="https://user-images.githubusercontent.com/26217480/70808925-be179700-1d8e-11ea-967a-9d1c84eff57c.PNG" width="60%"/>
+* Add support to allow Material entities to specify face culling method, support GLTF doubleSided property. (#15)
+* Fix wearables not disappearing with avatar. (#15)
+
+#### Technicals
+
+* Updated CMake's SDL2 file to build correctly on Windows and Linux. (#3, #20, #21)
+* Fix audio failure due to missing VR. (#31)
+* VideoDecodeStats folder added to .gitignore. (#2)
+* Fix crash on secondary camera usage. (#32)
+* Fix QMetaObject::invokeMethod crash when trying to bind menu. (#39)
+* Add experimental Ubuntu 19.10 build support. (#53)
+* VCPKG modernization. (#53)
+* Fix Linux Building with `export HIFI_MEMORY_DEBUGGING=1`. (#54)
+* Fix avatar walk speed per avatar scaling. (#55)
+* Fix CMake build failing due to Public Key on VCPKG. (#56)
+* Fix thousands of 'is too small to hold all values' warnings on Linux (#58)
+* Added Opus Audio Codec support. (#86)
+* Fix running scripts in Interface from being cleared occasionally. (#162)
+* Always display vertical scrollbar in Settings dialogs that scroll. (#113)
+* GitHub CI work-in-progress. (#51)
+* Added ArrayBuffer support to websockets. (#66)
+* Fix Clara.io type error issue. (#67)
+* Make default copy constructors explicit. (#104)
+* Fix updated TBB library hash in vcpkg portfile. (#222)
+
+
+## Server
+
+The numbers at the end of each item are the PR numbers in the Project Athena [GitHub repo](https://github.com/kasenvr/project-athena/pulls?utf8=%E2%9C%93&q=milestone%3Av0860-k2).
+
+#### General
+
+* Added environment variables to change domain server ports. (#15)
+* Fix server log spam `///`. (#175)
+
+
+## Documentation
+
+#### User Docs
+
+The numbers at the end of each item are the PR numbers in the Athena-Docs-Sphinx [GitHub repo](https://github.com/kasenvr/athena-docs-sphinx). 
+
+* Added initial shader documentation that can be found under Create > Materials, or [here](https://docs.projectathena.dev/create/materials/procedural-shaders.html). (#17)
+* Various other updates and fixes to the main documentation. (#1, #2, #3, #4, #5, #9, #10, #12, #23)
```

Comment:
```suggestion
* Various other updates and fixes to the main documentation. (#1, #2, #3, #4, #5, #9, #10, #12)
```
#23 isn't merged; it will be in K3 with product name changes.